File: demo/springboot/demo/src/main/java/com/demo/domain/dto/http/resonse/OssFileNameUpdateResponse.java
package com.demo.domain.dto.http.resonse;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;

@Data
public class OssFileNameUpdateResponse {
    @JsonFormat(shape = JsonFormat.Shape.STRING)
    private Long fileId;
    private String newFileName;
    private Boolean updated;
    private String message;
}

File: demo/app/app/src/main/java/com/vectordemo/domain/dto/http/response/OssFileNameUpdateResponse.kt
package com.vectordemo.domain.dto.http.response

import com.google.gson.annotations.JsonAdapter
import com.vectordemo.utils.json.WireUserIdJsonDeserializer

data class OssFileNameUpdateResponse(
    @JsonAdapter(WireUserIdJsonDeserializer::class)
    val fileId: String? = null,
    val newFileName: String? = null,
    val updated: Boolean? = null,
    val message: String? = null
)
